components/org.wso2.carbon.identity.application.authenticator.biometric.servlet/src/main/java/org/wso2/carbon/identity/application/authenticator/biometric/servlet/internal/BiometricEndpointServiceComponent.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package org.wso2.carbon.identity.application.authenticator.biometric.servlet.internal;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.eclipse.equinox.http.helper.ContextPathServletAdaptor;
+import org.osgi.service.component.ComponentContext;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.component.annotations.ReferenceCardinality;
+import org.osgi.service.component.annotations.ReferencePolicy;
+import org.osgi.service.http.HttpService;
+import org.wso2.carbon.identity.application.authenticator.biometric.servlet.servlet.BiometricServlet;
+import javax.servlet.Servlet;
+
+/**
+ * Service component class for the SAML SSO service.
+ */
+@Component(
+        name = "identity.sso.saml.component",
+        immediate = true)
+public class BiometricEndpointServiceComponent {
+
+    private static final Log log = LogFactory.getLog(BiometricEndpointServiceComponent.class);
+    private HttpService httpService;
+
+    @Activate
+    protected void activate(ComponentContext ctxt) {
+        log.info(" in the servlet internal");
+
+        String biometricEndpoint = "/samlbiomtriccheck";
+        Servlet samlBiometricServlet = new ContextPathServletAdaptor(new BiometricServlet(), biometricEndpoint);
+        try {
+            httpService.registerServlet(biometricEndpoint, samlBiometricServlet, null, null);
+        } catch (Exception e) {
+            String errorMsg = "Error when registering the new Biometric Endpoint via the HttpService.";
+            log.error(errorMsg, e);
```

Comment:
Please check whether if error message and stack trace is printed twice(since we throw the exception and logging as an error at the same time). If that's the case, we can remove logging.